project/CONTRIBUTING.md

```diff
@@ -271,6 +271,18 @@ or postponed to next major release if BC is not possible.
 
 If you are not sure what should be done, don't hesitate to open an issue about your PR project.
 
+##### Dependencies changes
+
+If you want to change some dependencies, here is the rules:
+
+- Don't add version support lower than the current one.
+- Don't change the highest version support to a lower one.
+- Lower version dropping is accepted as a Backward Compatible change according to [semver](semver_dependencies_update),
```

Comment:
Shouldn't it be with square brackets everywhere? `[semver][semver_dependencies_update]` ?

Reply:
Indeed.